src/Components/Form/FormFields/FormField.tsx

```diff
@@ -71,7 +71,12 @@ const FormField = ({
         )}
       </div>
       <div className={field?.className}>{children}</div>
-      <FieldErrorText error={field?.error} className={field?.errorClassName} />
+      {field?.error ? (
+        <FieldErrorText
+          error={field?.error}
+          className={field?.errorClassName}
+        />
+      ) : null}
```

Comment:
Revert this, as this now breaks the consistent padding in all other places

<img width="1322" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/1426c8cb-197e-4248-ac18-ae677916ff7e">


```suggestion
      <FieldErrorText error={field?.error} className={field?.errorClassName} />
```